docs/UserGuide.md

```diff
@@ -635,6 +635,9 @@ If your changes to the data file makes its format invalid, Swift+ will discard a
 **Q**: How do I transfer my data to another computer?<br>
 **A**: Install Swift+ in the other computer. Replace the default data file `addressbook.json` with the file that contains the data of your previous Swift+.
 
+**Q**: Why pressing Tab does not autocomplete the whole command?<br>
```

Comment:
Please replace with
`Why does autocomplete not complete the whole suggestion after pressing Tab?`